### PR TITLE
Developer Content Tweaks

### PR DESCRIPTION
**Pull request by flynnduism**
This PR makes a couple of content updates and modifies the UI buttons to better promote Spin.

**Homepage Content**

* update the intro text to be more descriptive
* add an event (KubeCon) so slider isn't blank
* update social channels list

**Nav CTAs**

dev-nav

These buttons have been focused on Cloud, regardless of what project docs you are viewing. And we should do a better job at encouraging more Spin ⭐ 's.

This PR will default the menu to show Spin actions (Star on Github, go to Quickstart) for the homepage and `/spin` sections. The Cloud login/signup buttons will still appear if you are looking at `/cloud/` pages.

### CODE REVIEW

File: templates/content_navbar.hbs [hunk 1]
```diff
@@ -144,12 +145,50 @@
...
+                href="https://github.com/fermyon/spin"
+                data-icon="octicon-star"
+                data-size="large"
+                data-show-count="true"
+                data-color-scheme="no-preference: light; light: light; dark: dark;"
+                aria-label="Star fermyon/spin on GitHub">Star</a>
+                <script async defer src="https://buttons.github.io/buttons.js"></script>
+            </span>
+            <a class="navbar-item button is-primary is-rounded is-medium plausible-event-name=cta-spin-quickstart" href="https://developer.fermyon.com/spin/quickstart" target="_blank" aria-label="menuitem">
+              <img src="/static/image/spin-icon.svg" alt=" " />
```
Comment (karthik2804) [inline templates/content_navbar.hbs:161]: seems to be a broken image link

**Comment by karthik2804** [comment]
The star button seems to be stuck in dark mode on chrome and firefox for me 
image

File: templates/content_navbar.hbs (hunk 1 above)
Comment (flynnduism) [reply templates/content_navbar.hbs]: Removed these

**Comment by flynnduism** [comment]
> The star button seems to be stuck in dark mode on chrome and firefox for me

Struggling to recreate this (Chrome, FIrefox). @karthik2804 could you delete cookies (saved theme pref) and re-test? 🥺

**Comment by karthik2804** [comment]
@flynnduism I took at peek at the CSS, and it seems to be directly styling to the default system theme. The element has the following CSS applied to it which mean it switches based on my system preference instead of the theme. Possibly some `!important` will solve it 😅 
image

**Comment by flynnduism** [comment]
> @flynnduism I took at peek at the CSS, and it seems to be directly styling to the default system theme. The element has the following CSS applied to it which mean it switches based on my system preference instead of the theme. Possibly some `!important` will solve it 😅

# GitHub Button Theme

light-dark-gh-brtn

Following up on this - the Github button theme is not easy to fix! 

Screen Shot 2024-08-23 at 3 10 13 PM

`!important` doesn't cut it, and reinforcing with `(prefers-color-scheme: light/dark)` is not fixing it either. The button is rendered after the DOM, and is treated like an iframe (shadow root) - our local styles and overwrite attempts have no effect.

# Blocking?

I'd suggest filing the button's CSS theme inheritance as an issue, and try and get these content updates out to the live site. What do you think @karthik2804

**Comment by karthik2804** [comment]
It is definitely not blocking. Can be added as an issue.

**Review by karthik2804 — APPROVED** [review]

**Pull request merged**